penguin/run_mode.py

```
@@ -657,14 +657,44 @@ async def start_continuous(
                         logger.debug(
                             "No project-scoped tasks available; refusing synthetic drift task"
                         )
+                        blocked_candidates = []
+                        try:
+                            blocked_candidates = (
+                                await self.core.project_manager.get_blocked_ready_candidates_async(project_id)
+                            )
+                        except Exception as blocked_err:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)